### PR TITLE
Collect analytics

### PR DESCRIPTION
**Pull request by xtina-starr**
Addresses https://artsyproduct.atlassian.net/browse/GROW-884.

screen shot 2018-09-12 at 2 58 01 pm

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 1da47e2cb8bd269c9ead806abdc5bc12edb9ff64

https://deploy-preview-1238--artsy-reaction.netlify.com

File: src/Apps/Collect/AnalyticsProvider.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  render() {
+    const {
+      __fragments,
+      __id,
+      Component,
+      tracking,
+      ...remainingProps
+    } = this.props
+
+    console.log("tracking:", tracking)
```
Comment (l2succes) [inline src/Apps/Collect/AnalyticsProvider.tsx]: trailing console.log

File: src/Apps/Collect/FilterState.tsx [hunk 2]
```diff
@@ -36,9 +38,12 @@ const initialState = {
 
 export class FilterState extends Container<State> {
   state = cloneDeep(initialState)
+  tracking: any
```
Comment (l2succes) [inline src/Apps/Collect/FilterState.tsx]: pretty sure we have an interface for `tracking` we can type this here

File: src/Apps/Collect/AnalyticsProvider.tsx (hunk 1 above)
Comment (xtina-starr) [reply src/Apps/Collect/AnalyticsProvider.tsx]: Ah good catch!

File: src/Apps/Collect/FilterState.tsx (hunk 2 above)
Comment (xtina-starr) [reply src/Apps/Collect/FilterState.tsx]: I thought so too and looked through the project and so that it was pretty consistently typed as `any`.

**Comment by l2succes** [comment]
@xtina-starr There's some failing tests, Let's merge after getting the build green

**Pull request merged**